bin/factorial/handleArgument.js

```diff
@@ -0,0 +1,31 @@
+import { destructureArgument, validateArgument } from "../shared/utils.js";
+
+export const handleArgument = cliArguments => {
+  let type = "";
+  let method = "";
+  let n = 0;
+
+  for (let argument of cliArguments) {
+    if (!validateArgument(argument)) {
+      throw new Error(`invalid argument ${argument}`);
```

Comment:
You could make a better exception throwing by setting a specific `Error` instance rather than just a basic `Error`. See details here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#error_objects.